test/sw_mgmt_spec.rb

```diff
@@ -46,6 +46,24 @@
     }
   end
 
+  describe ".init" do
+    it "initializes package management and returns true" do
```

Comment:
This test basically mimics the implementation line by line. There is no much value in such a test.